server.go

```diff
@@ -318,11 +321,13 @@ func basicAuth(h http.HandlerFunc) http.HandlerFunc {
 		h.ServeHTTP(w, r)
 	}
 }
+// Comment
 
 //------------------------------------------------------------------------------
 //http handlers
 func GetUser(w http.ResponseWriter, r *http.Request) {
 	_, login, _ := getCredentials(r)
+	// Comment
```

Comment:
Shouldn't we check the `bool` returned by the function: https://github.com/vojapet/team-manager/blob/master/server.go#L274?

Reply:
Generally speaking yes. On one hand in this very case we know that the user already pass the authentication so the credential should be there. On the other hand there could be "race condition" if some other request delete the user between those two calls. Since there is no delete function implemented I omit the check. Adding the check here makes no harm and add readability - so yes I understand the point here.